packages/website/docs/sources.md

```diff
@@ -236,25 +260,158 @@ const autocomplete = createAutocomplete({
 });
 ```
 
-This pattern can be used to store data in the [context](context) before returning the sources.
+:::note
+
+You can use the official [`autocomplete-plugin-query-suggestions`](createQuerySuggestionsPlugin) plugin to retrieve Query Suggestions from Algolia.
+
+:::
+
+### Customizing items with templates
+
+In addition to defining data sources for items, a source also lets you customize how to display items using [`templates`](#templates-specific-to-algoliaautocomplete-js).
+
+:::info
+
+Templates are only supported in [autocomplete-js](autocomplete-js).
+
+:::
```

Comment:
This is [what the document suggests later on](https://docs-1--algolia-autocomplete.netlify.app/docs/sources/#templates-specific-to-algoliaautocomplete-js). However, there's [another example](https://docs-1--algolia-autocomplete.netlify.app/docs/state#listening-to-state-changes) where `templates` is being used with `createAutocomplete`.

Is any of those outdated?

Reply:
Good catch, that's an error.

I think we should default to documenting `@algolia/autocomplete-js` instead of `core`. It'll be less confusing and only power users will need `core`. Wdyt?

Reply:
So the error is that both `@algolia/autocomplete-core` and `@algolia/autocomplete-js` accept `templates`, right?

I agree that we should stick to `@algolia/autocomplete-js` examples as much as possible (not just for this case, but for the docs overall) unless we're specifically documenting `core`.

Reply:
The templates concept only exists in `@algolia/autocomplete-js` so the error is to have used `createAutocomplete()` instead of `autocomplete()`.